javascript-ajax/users.js

```diff
@@ -0,0 +1,19 @@
+
+var $userList = document.querySelector('#user-list');
+var xhr = new XMLHttpRequest();
+xhr.open('GET', 'https://jsonplaceholder.typicode.com/users');
+xhr.responseType = 'json';
+xhr.addEventListener('load', function () {
+  console.log(xhr.status);
+  console.log(xhr.response);
+  for (var i = 0; i < xhr.response.length; i++) {
+    if (xhr.response[i]) {
```

Comment:
This conditional is not necessary, you are looping to the length of the array, so you will only see items that exist in the array